composer.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "woocommerce/snapchat-for-woocommerce",
+	"description": "Snapchat for WooCommerce",
+	"license": "GPL-3.0-or-later",
+	"autoload": {
+		"psr-4": {
+			"SnapchatForWoocommerce\\": "includes/"
+		}
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^9.0",
+		"automattic/jetpack-autoloader": "^2",
```

Comment:
Not sure there's a benefit to using the Jetpack autoloader over just a normal composer autoloader?